ports/marzbanpp/portfile.cmake

```
@@ -0,0 +1,22 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO andrascii/marzbanpp
+    REF "1.0.1"
+    SHA512 8db3ca283d119efadf9f9ba5d573263c92402a2dda82683499918a5889ddff502a0bcff0658a26b927f2d3a373e42dbcf59cf7aad2fab6313210eb39fff78b00
+    HEAD_REF marzbanpp
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(PACKAGE_NAME "marzbanpp")
```

Default is port name.
```suggestion
vcpkg_cmake_config_fixup()
```